### PR TITLE
chore: fix broken ui in fast-website, various content fixes, update to use new color vars

### PR DESCRIPTION
**Pull request by chrisdholt**
# Description

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/website/src/docs/api/fast-foundation.api.json [hunk 1]
```diff
@@ -40,7 +40,11 @@
             {
               "kind": "Property",
               "canonicalReference": "@microsoft/fast-foundation!Accordion#expandmode:member",
+<<<<<<< HEAD
```
Comment (awentzel) [inline sites/website/src/docs/api/fast-foundation.api.json:43]: merge conflicts?

Comment (chrisdholt) [reply sites/website/src/docs/api/fast-foundation.api.json:43]: I'll generate this again - it's an automated thing so this will get resolved

Comment (chrisdholt) [reply sites/website/src/docs/api/fast-foundation.api.json]: @awentzel, should be fixed.

**Comment by khamudom** [comment]
Not sure what changes could cause this. But if you add `align-self: end;` to `.github-link` css selector, in the `@media screen and (max-width: 800px)` it will fix it.
image

**Comment by chrisdholt** [comment]
> Not sure what changes could cause this. But if you add `align-self: end;` to `.github-link` css selector, in styles.css, under `@media screen and (max-width: 800px)` it will fix it.
> image

Thanks for smoking @khamudom - can we take that as a follow-up? There's quite a few things we identified from a fit and finish standpoint that needs to be addressed with the header at non-responsive viewports as well. We'll likely pick up after Wednesday.

**Pull request merged**